### DISCUSSION TITLE
Does shields.io support obtaining versions from nexus pypi repo?

### DISCUSSION THREAD

**Discussion by Dzordzu** [badges/shields#6045] [Q&A] [answered]
I've been trying to get version of my python package from the nexus repo. The problem seems to be in the 'group'. This property (for pypi) is set to null. Unfortunately but none of the nexus badges support empty value for 'group;. 
Is there any option to achieve my target?

**Comment by PyvesB** [comment]
Hello @Dzordzu, sorry we missed this question. Do you have an example of public Nexus repository that hosts Python packages?

**Reply by Dzordzu** [reply]
Yep. For example [my repo](https://nexus.dzordzu.pl/#browse/browse:python-de2sert)

**Comment by Dzordzu** [comment]
The following issues may happen if anyone (including me) will be willing try to implement direct search

1. For the non maven repos, LATEST tag is not guaranteed to work (you should be using semver, or tail last record in order to achieve the wanted result)

**Comment by PyvesB** [comment, marked as answer]
Our Nexus badges support overriding the query options that are being sent to the server. You can get badges for your pypi repo using URLs similar to the following:
https://img.shields.io/nexus/r/-/de2sert-pyparse?queryOpt=%3Agroup%3D&server=https%3A%2F%2Fnexus.dzordzu.pl&nexusVersion=3
![](https://img.shields.io/nexus/r/-/de2sert-pyparse?queryOpt=%3Agroup%3D&server=https%3A%2F%2Fnexus.dzordzu.pl&nexusVersion=3)

Note in particular the `queryOpt=%3Agroup%3D` query parameter which basically nullifies the `-` group
